### PR TITLE
share tools

### PR DESCRIPTION
**Pull request by collin**
Look ma, `<share-tools>`

These need some documentation love.

### CODE REVIEW

File: elements/share-tools/share-tools.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+}
+
+%share-tool {
+  background-color: black;
+  display: inline-flex;
+  font-size: 1.5em;
+
+  .share-button-icon {
+    color: white;
```
Comment (jmelvnsn) [inline elements/share-tools/share-tools.scss]: Please to alphabetize the styles

**Comment by jmelvnsn** [comment]
giphy

**Comment by collin** [comment]
@kand @spra85 @daytonn @shawncook 

https://github.com/theonion/bulbs-elements/tree/share-tools/elements/share-tools
👆Docs

File: elements/share-tools/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# `<share-tools>`
```
Comment (kand) [inline elements/share-tools/README.md]: Follow a more consistent formatting here, a single large header, and everything else nested.

**Comment by spra85** [comment]
👍

**Comment by kand** [comment]
👍

**Comment by jmelvnsn** [comment]
You deserve an award for this one: 

sdfxd

**Pull request merged**